### ISSUE TITLE
Add CMIP6Plus mip era

### ISSUE THREAD

**Issue by durack1** [PCMDI/pcmdi.github.io#530]
Adding CMIP6Plus mip era, so that termsofuse and other placeholders can be populated

**Comment by taylor13** [comment]
I think we should consider renaming mip_era with something referring to infrastructure/data specifications/metadata requirements (e.g. "data_specs").   In the future it would be nice if the data_specs didn't change as the experiment design evolves.  For the DECK, especially, this would  I think we may find some MIPs running old versions of historical expts., for example, with new models.  These would be considered to be "old era" experiments, but they might conform to data_specs.

**Comment by durack1** [comment]
@taylor13 agreed. The recognition that "specs" are more meaningful than "era" is clear, this tweak is just to get the CMIP6Plus project rolling, and we should do a cleanup of these identifiers as we evolve through to CMIP7

**Comment by taylor13** [comment]
sounds good.

**Comment by durack1** [comment]
> @taylor13 agreed. The recognition that "specs" are more meaningful than "era" is clear, this tweak is just to get the CMIP6Plus project rolling, and we should do a cleanup of these identifiers as we evolve through to CMIP7

Thinking a little more about this, for a CMIP7 framing, switching from "CMIP era" to "infrastructure/specs era" makes sense to me, and is a fair bit more disruptive, so would take more time to document and articulate. I think the path that we're on with CMIP6Plus and collapsing many of the pieces into a centrally, community-managed format will be what we need to make such a large change, but would also require the projects/MIPs that are using these to select their "era" and so the **project_CVs** templates that we are going with work well that way

**Comment by durack1** [comment]
@taylor13 see anything wrong with https://pcmdi.llnl.gov/CMIP6Plus/TermsOfUse/TermsOfUse6-5.html - it's just been stood up, as it's likely the shortest path to fill holes in the current mip-tables/CVs assortment

**Comment by taylor13** [comment]
I've sent @durack1  some suggested edits by email.

**Comment by taylor13** [comment]
In the CMIP6plus terms of use guidance, the first bullet point is:

"Adhere to licensing restrictions associated with the Creative Commons Attribution 4.0 International CC BY 4.0 license."

but some models might impose a different license according to the statement 

"The licensing agreements governing CMIP6Plus data depend on the model but generally conform to a standard template (nb.: the following is just an example and does not apply to all model output):"

If there might be licenses different from "CC BY 4.0", then the first bullet point should somehow be qualified.  Perhaps it could read,  

"adhere to licensing restrictions associated with the Creative Commons Attribution 4.0 International CC BY 4.0 license and any more restrictive license restrictions that may have been imposed by some models (see *LINK*)".  

"Link" would point to the CV where the license restrictions for each model have been recorded.

**Comment by durack1** [comment]
Refers to details in https://pcmdi.llnl.gov/CMIP6Plus/TermsOfUse/TermsOfUse6-5.html - will reopen.

@wolfiex is there another CMIP6Plus page that we need to consider syncing?
